CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The appc spec version has been updated to v0.7.4
 - CPU and memory resource isolators can be specified on the command line to override the limits specified in the image manifest ([#1851](https://github.com/coreos/rkt/pull/1851), [#1874](https://github.com/coreos/rkt/pull/1874)). See rkt's [overriding isolators](https://github.com/coreos/rkt/blob/master/Documentation/subcommands/run.md#overriding-isolators) documentation.
 - CPU and memory resource isolators can now be used within the kvm stage1 ([#1404](https://github.com/coreos/rkt/pull/1404))
 - The `rkt image list` command can now display the image size ([#1865](https://github.com/coreos/rkt/pull/1865)).
+- A new stage1 flavor has been added: fly; and it represents the first experimental implementation of the upcoming rkt fly feature. ([#1833](https://github.com/coreos/rkt/pull/1833))
```

Comment:
Does it worth mentioning in the top paragraph? ;)

No need to do it in this PR. I can add it in the release PR if you suggest something.